### ISSUE TITLE
pull our own awx images

### ISSUE THREAD

**Issue by g18z0n** [rhythmictech/terraform-aws-awx#18]
I recently found your project and it looks very promising!

The only concern i'm having is the fact that you are pulling your own custom images of awx_task and awx_web from your repo, wich in many use cases, this process is not "transparent" enough to be used.

Is there any chance that you could publish the changes needed for the official images to work with your terraform implementation? so we are able to use our own custom images.

This would be also good for faster upgrades, since you are currently pulling version 6 and version 7 is already available with some good features as SSL support.

**Comment by sblack4** [comment]
Of course! Everything you need to build the images yourself is here, https://github.com/sblack4/awx_containers. 

I'll variable-ize the `image` value in the service spec and you'll be able to push your own images without modifying the repo.

The only major change was my adding the ability to pull the `SECRET_KEY` from the environment.

**Comment by sblack4** [comment]
I started this but haven't had time to finish it, https://github.com/rhythmictech/terraform-aws-awx/tree/custom-images-%2318

It really just involves changing the `image` value in the `templates` json files.
